WeightProducer/python/MCSampleValues.py

```diff
@@ -1018,6 +1018,10 @@ class MCSampleValuesHelper():
         "RPV_2t6j_mStop-1400_mN1-100" : {
             "CrossSection" : XSValues(XS_13TeV=0.000473),
         },
+	"exoticVLQtbb_MiniAOD_1TeV" : {
+	    "CrossSection" : XSValues(XS_13TeV=0.0397),
+	},
```

Comment:
You should have a cross section value for every signal. It changes as you change the mass.